bee-common/bee-common/Cargo.toml

```diff
@@ -12,16 +12,19 @@ homepage = "https://www.iota.org"
 
 [dependencies]
 chrono = "0.4"
+console-subscriber = { git = "https://github.com/tokio-rs/console.git", branch = "main", optional = true }
 fern = { version = "0.6", features = [ "colored" ] }
 jsonwebtoken = { version = "7.2", optional = true }
 log = { version = "0.4", features = [ "serde" ] }
 rand = { version = "0.8", optional = true }
 rust-argon2 = { version = "0.8", optional = true }
 serde = { version = "1.0", features = [ "derive" ] }
 thiserror = "1.0"
+tracing-subscriber = { version = "0.2.21", optional = true }
```

Comment:
Should be safe to go without the patch version here as well, right?

Reply:
Yes please, remove it